app/controllers/charges_controller.rb

```diff
@@ -0,0 +1,45 @@
+class ChargesController < ApplicationController
+  def create
+    # @order = Order.find(params[:order_id])
+    total = cart.total_price
+    total_in_cents = total * 100
+    # @order.update(status: "paid")
+
+    # Amount in cents
+    @amount = total_in_cents.to_i
+
+    customer = Stripe::Customer.create(
+      :email => current_user.email,
+      :card  => params[:stripeToken]
+    )
+
+    charge = Stripe::Charge.create(
+      :customer    => customer.id,
+      :amount      => @amount,
+      :description => Order.last.id + 1,
+      :currency    => 'usd'
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.